docs/UserGuide.md

```diff
@@ -131,6 +130,33 @@ Examples:
 * `find -sp n/Alex s/Orthopaedic` returns any specialists names including the string `Alex` who has the `Orthopaedic` specialty
 <br>
 
+### Editing a pre-existing patient or specialist record: `edit`
```

Comment:
Great update of the user guide! Would be good if you could update the command summary table at the bottom as well.

Reply:
Thanks for pointing that out. I added the edit command into the command summary table.